src/theme/components/skipNav.ts

```diff
@@ -0,0 +1,23 @@
+const SkipNav = {
+  baseStyle: {
+    // Don't display links by default...
+    a: {
+      backgroundColor: "ui.white",
+      height: "1px",
+      left: "-10000px",
+      overflow: "hidden",
+      position: "absolute",
+      top: "auto",
+      width: "1px",
+      // Only display when the user focuses on the links.
+      _focus: {
```

Comment:
Let’s get this a little closer to the styles as it is currently implemented.  Add the following:
```
padding: var(--nypl-space-inset-extranarrow) var(--nypl-space-inset-narrow),
border: 1px solid var(—nypl-colors-ui-gray-dark)
```